charts/emissary-ingress/values.yaml.in

```diff
@@ -52,16 +52,17 @@ autoscaling: # +doc-gen:break
   metrics:
   - type: Resource
     resource:
-      name: cpu
+      name: memory
       target:
         type: Utilization
-        averageUtilization: 60
+        averageUtilization: 180
```

Comment:
Kubernetes uses the `request` values to base its metrics off. In the case of emissary, I think you've well-tuned the requests to be what an unused emissary pod would need. These value changes make the HPA behave better for what a user would expect - that the HPA scales up when approaching the **limit**. But, the kubernetes HPA doesn't take into account the limit settings.

It's perfectly OK to have the HPA use a value greater than 100% - maybe unintuitive - but ok. Of course, these values only hold when using the default values below for cpu/memory resource requests/limits.
But, if a user is changing those, then it wouldn't be surprising that they would have to change these settings as well.

For me, it *was* surprising that I would have to change these values, when I *didn't* change the resources.